src/Containers/Event/Event.js

```diff
@@ -72,6 +75,31 @@ const Event = (props) => {
     return(<p key={key}>{label}</p>)
   });
 
+  const checkIfRSVP = () => {
+    const checkRSVP = props.userRSVPs.filter(eventID => props.id === eventID);
+    if (!!checkRSVP.length){
+      return (
+        <button onClick={()=> props.removeRSVP(props.id)}>Remove RSVP</button>
+      )
+    } else {
+      return (
+        <button onClick={()=> props.addRSVP(props.id)}>RSVP</button>
+      )
+    }
+  };
+
```

Comment:
I got so stuck on a method like this for a mod3 project but great job this looks awesome.